www/templates/html/MediaPlayer.tpl.php

```diff
@@ -4,15 +4,175 @@
+
+                                $transcript.find("li").addClass("highlight");
+
+                            });
+
+                            $captionSearch.find(".mh-parse-caption").on("keyup focus", function(e){
+
+                                e.stopPropagation();
+
+                                var search = $(this).val().toLowerCase();
+                                var subtitlesLength;
+                                var i;
+
+                                //$transcript.find("li").removeClass("highlight");
+                                if (!search) {
+                                    return;
+                                }
+
+                                subtitlesLength = track.entries.text.length;
+
+                                for (i = 0; i < subtitlesLength; i++) {
+
+                                    var line = track.entries.text[i].toLowerCase();
+
+                                    if (line.indexOf(search) > -1){
+
+                                        $transcript.find("li").eq(i).addClass("highlight");
+
+                                    }else{
+                                        $transcript.find("li").eq(i).removeClass("highlight");
+                                    }
+                                };
+
+                            });
+
+                            t.container.find(".mh-paragraph-icons").on("click", function(){
+
+                                t.container.find(".mh-caption-search").toggleClass("bulleted");
+
+                            });
+
+                            $transcript.on('click', 'li', function() {
+                                var time;
+                                time = $(this).data('timeOffset')
+
+                                if (!time) {
+                                    return;
+                                }
+
+                                t.setCurrentTime(time);
+                            });
+
+                            var listItems = [];
+
+                            for (var i = 0; i < track.entries.text.length; i++) {
+
+                                //var subtitles = track.entries.text[i].replace(/<(?:.|\n)*?>/gm, '');
+
+                                listItems.push($('<li>',  {"class": "highlight"})
+                                    .data('timeOffset', track.entries.times[i].start)
+                                    .text(track.entries.text[i])
+                                    .prepend($('<span>').text('[' + displaytime(track.entries.times[i].start*1000) + '] '))
+                                );
+                            };
+
+                            $transcript.append(listItems);
+
+                        };
+
+                        var origsSetTrack = t.enableTrackButton;
+                        t.enableTrackButton = function(e) {
+                            origsSetTrack.call(this, e);
+                            setTranscript(t.tracks[0]);
```

Comment:
If the use case doesn't work with `setTrack` you should avoid the assumption that the `[0]` is the correct one to load.